src/types/enum.rs

```diff
@@ -0,0 +1,3 @@
+pub struct Enum;
```

Comment:
I think this file is dead code which can be thrown away

Reply:
I didn't see that. Will do.